src/test/java/net/phptravels/WebLogin.java

```diff
@@ -13,50 +13,39 @@
 public class WebLogin implements IAbstractTest {
```

Comment:
Class name should reflect what tests it contains
You don't have here login tests